luxonis_ml/data/parsers/coco_parser.py

```diff
@@ -216,6 +217,10 @@ def generator() -> DatasetIterator:
 
                     if "keypoints" in ann.keys():
                         kpts = np.array(ann["keypoints"]).reshape(-1, 3)
+                        # clip values inplace
+                        np.clip(kpts[:, 0], 0, img_w, out=kpts[:, 0])
+                        np.clip(kpts[:, 1], 0, img_h, out=kpts[:, 1])
```

Comment:
Do we always want to clip keypoints?

Reply:
Yes, otherwise KeypointAnnotation fails since it expects [tuple of normalized floats](https://github.com/luxonis/luxonis-ml/blob/f3a94195a73a012e4c18d021dc9417b187ffd794/luxonis_ml/data/datasets/annotation.py#L165).

Reply:
Hm, maybe it would make more sense to allow the keypoints to be in [-1, 1] range instead, but I'm not entirely sure what's the best approach here.